docker/build.sh

```diff
@@ -2,10 +2,12 @@
 
 set -e
 
-echo "Build binary using golang docker image"
-docker run --rm -ti \
-    -v "`pwd`":/go/src/github.com/restic/restic \
-    -w /go/src/github.com/restic/restic golang:1.16-alpine go run build.go
+export DOCKER_BUILDKIT=${DOCKER_BUILDKIT-1}
 
 echo "Build docker image restic/restic:latest"
-docker build --rm -t restic/restic:latest -f docker/Dockerfile .
+docker build \
+  --rm \
+  --pull \
+  --file docker/Dockerfile \
+  --tag DOCKER_TAG-restic/restic:latest \
```

Comment:
What is the purpose of the `DOCKER_TAG-` prefix?

Reply:
This was a left-over from an idea I head to make the tag configurable via env variables. But I haven't cleaned up properly.